controllers/articles.controller.js

```diff
@@ -11,4 +11,17 @@ exports.getArticleByID = async (req, res, next) => {
     }catch(error){
         next(error)
     }
-}
\ No newline at end of file
+}
+exports.getAllArticles = async (req, res, next) => {
+    try {
+        const articles = await selectAllArticles()
+    
+        const formattedArticles = articles.map(article => {
+            const {body, ...rest} = article
+            return rest
```

Comment:
Please change this ^^^ 
We should not be removing the things we dont want after fetching them. Your SQL needs to exclude the body if you don't want to fetch it 🙂